### ISSUE TITLE
Edit content field and remove type field from board.yml credits file

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#2794]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, I want to edit the content field and remove the type field from a credit's yml file so that redundant code is removed and the code is easier to understand.

### Details
Currently, in each credit's yml file, there is a `content` field and a `type` field, which contains similar information. Since the fields are redundant, we decided to remove the `type` field.  

Also, we changed the `content` field to the `content-type` field to make it clearer what its purpose is. The possible values for `content-type` field are image, video, or audio; other types of content can be added if necessary. In the future, this will allow developers a way to differentiate easily between different types of content in order to show each credit's media file correctly on the website.

### Action Items
For the file `_data/internal/credits/board.yml`, do the following:
- [ ] Change line 4 for the content field from
  ```
  content: icon
  ```
  to
  ```
  content-type: image
  ```
- [ ] Remove line 11 for the type field
  ```
  type: icon
  ```
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #2775 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #2775 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is tracked in the epic #2775.

#### File and code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_data/internal/credits/board.yml`

- Click on the links below to see the source file code:
https://github.com/hackforla/website/blob/0a6a6e08f231f0fd5ac2c7c9222a6bf6b8da682a/_data/internal/credits/board.yml#L4
https://github.com/hackforla/website/blob/0a6a6e08f231f0fd5ac2c7c9222a6bf6b8da682a/_data/internal/credits/board.yml#L11

**Comment by Olanrewaju-Ak** [comment]
ETA: End of the day on 5/24
Availability: 1 hours
